src/HideLooseAssetService.cpp

```diff
@@ -19,12 +22,69 @@ namespace BsaPacker
 			return false;
 		}
 
+		// Get QStringList of blacklisted extensions and remove the . so we can match with QFileInfo::suffix later
```

Comment:
Add comment:
`Only hide loose files that were not excluded when creating the Archive, so might still be needed by the mod`